__tests__/supertest.test.js

```diff
@@ -91,3 +91,47 @@ describe("GET /api/reviews", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+  it("should return an array of comment objects for the given review_id with the following properties: comment_id, votes, created_at, author, body, review_id", () => {
+    return request(app)
+      .get("/api/reviews/3/comments")
+      .expect(200)
+      .then((res) => {
+        const comments = res.body.comments;
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            review_id: expect.any(Number),
+          });
+        });
+        expect(comments).toBeSorted("created_at", {
+          descending: true,
+        });
+        expect(comments.length).toBe(3);
+      });
+  });
+  it("no comments for given review id, responds with a 204", () => {
```

Comment:
Technically this should still be a 200 as a 204 is 'no content'. There is technically content there, it's just returning an empty array. That's still content. I know that seems weird but hopefully that sort of makes sense :)